src/Game.hxx

```diff
@@ -120,10 +121,7 @@ private:
     template<typename ArgumentType>
     void operator()(const ArgumentType&& event)
     {
-      string ErrorMsg = "Unimplemented Error: ";
-      ErrorMsg += __PRETTY_FUNCTION__;
-      ErrorMsg += " is not implemented";
-      throw RuntimeError(ErrorMsg);
+      throw UnimplementedError(TRACE_INFO "Unimplemented Error");
```

Comment:
Should we move this template to a seperate file? idk if Game.hxx is the best place

Reply:
I can make a `Game.inl.hxx` file to hide implementations but templates are inherently inlined, so they must be part of a header file.

- [ ] Make `Game.inl.hxx`